crates/api/src/module.rs

```diff
@@ -24,6 +24,7 @@ fn into_global_type(gt: &wasmparser::GlobalType) -> GlobalType {
     GlobalType::new(into_valtype(&gt.content_type), mutability)
 }
 
+#[allow(clippy::trivially_copy_pass_by_ref)]
 fn into_valtype(ty: &wasmparser::Type) -> ValType {
```

Comment:
`wasmparser::Type` implements `Copy` and is a simple enum; is there a reason for passing it by reference here?

Reply:
@sunfishcode this function is used with `map` which requires `&T`.

Reply:
Ah, makes sense.